specification/response-specification.md

```diff
@@ -55,9 +55,16 @@ Below are the parameters that will be encrypted using AES256 encryption algorith
 | txnid | String | UUID txnid \( To be sent back from the request \) |
 | sessionid | String | Value of sessionid received in the ‘ecreq’ field in the request. |
 | userid | String | The AA user id |
+| srcref | Array consentHandleDetails | Consent handle details for the array of consents received in the ecreq ‘srcref’ field |
```

Comment:
As per the community review discussion, when the `requestorType` is `LSP`:

1. Response field `srcref` will be a primitive string array instead of an object, similar to **ecreq.srcref**.
2. AAs will return an array of only accepted consents' handle identifiers via `srcref` field.
3. Response field `status` will be marked as Success ('S'), even if the end-user approves only a subset of requested consents.
4. Similar to point 3, if `status` is marked Success, `errorCode` will be set to Success (`0`) as well.

Consequently, previously proposed `consentHandleDetails` object will no longer be required.